src/main/java/com/shippo/net/APIResource.java

```diff
@@ -37,6 +35,8 @@
 
 	public static final Gson GSON = new GsonBuilder()
 			.setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
+            // find a way to not access Batch here
```

Comment:
The indentation is done by Eclipse. Not sure why it renders terribly here. I changed the settings to use space instead of tab to avoid this.